microsoft-edge/extensions-chromium/publish/api/using-addons-api.md

```diff
@@ -1,24 +1,86 @@
 ---
-title: Using the Microsoft Edge Add-ons REST API
+title: Using the Update REST API for Microsoft Edge Add-ons
 description: REST endpoints to automate publishing updates to add-ons that are submitted to the Microsoft Edge Add-ons store.
 author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.service: microsoft-edge
 ms.subservice: extensions
-ms.date: 03/12/2024
+ms.date: 09/19/2024
 ---
-# Using the Microsoft Edge Add-ons REST API
+# Using the Update REST API for Microsoft Edge Add-ons
 
-To publish an extension, you first use Partner Center, and then optionally you can use this REST API to update the extension.
+To publish an extension, you first use Partner Center, and then optionally you can use the REST API to update the extension (or use Partner Center instead).
 
-The Microsoft Edge Add-ons API provides a set of REST endpoints for programmatically publishing updates to add-ons submitted to the Microsoft Edge Add-ons store.  You can use these REST endpoints to automate the process of uploading and publishing new versions of your add-ons to the Microsoft Edge Add-ons store.  You use the **Publish API** page at Partner Center to enable the API and begin working with these API endpoints.
-
-<!-- To publish an extension, you first use Partner Center, and then optionally you can use the REST API to update the extension (or use Partner Center instead). -->
+The Microsoft Edge Add-ons API provides a set of REST endpoints for programmatically publishing updates to add-ons submitted to the Microsoft Edge Add-ons store.  You can use these REST endpoints to automate the process of uploading and publishing new versions of your add-ons to the Microsoft Edge Add-ons store.  You use the **Publish API** page at Partner Center to first enable the API and then begin using these API endpoints.
 
 To submit suggestions and feedback, enter an [Issue about the Add-ons API](https://github.com/MicrosoftDocs/edge-developer/issues/new?title=[Add-ons%20API]).
 
 
+<!-- ------------------------------ -->
+#### Versions of the Update REST API
+
+As of September 6, 2024, both v1.1 and v1 of this Update REST API are supported.  Later, v1 will no longer be supported.  The date for ending v1 support is not yet finalized.
+
+
+<!-- ---------- -->
+###### Components used
+
+##### [v1.1](#tab/v1-1)
+
+v1.1 uses an API key.  The UI in Partner Center covers the API key.
+
+The REST endpoints use specific request headers for v1.1.
+
+##### [v1](#tab/v1)
+
+v1 requires sending an access token.<!-- todo: to an access token URL?  an access token that's provided by/at an access token URL? -->  The Partner Center UI shows the access token URL.<!-- todo: define/relate "access token URL" vs. "access token" -->
```

Comment:
define and relate "access token URL" & "access token".  do you use the url to retrieve the token?

Reply:
answer is in section
https://review.learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/api/using-addons-api?branch=pr-en-us-3269&tabs=v1#retrieve-the-access-token
Translation to plain language:
"You'll use the access token URL to get an access token, and then will use that access token in the header when using an API endpoint such as [Foo a bar](#foo), below."
...
"To get an access token, send a POST request to the access token URL, as follows:

i'll try that wording.  @bhuvanapriyap 

There's confusing, non-simple wording mixed-in like "the client credentials grant" and "tenant info" - non-introduced terms that have not been defined, making it hard to follow.  can we simply/ streamline the wording in plain language?

are those terms in the article's Glossary? no:
https://review.learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/api/using-addons-api?branch=pr-en-us-3269&tabs=v1#terminology

sentence is garbled, needs rewriting / resequencing:
"To get a token by using the client credentials grant, send a POST request to the Access token URL (the OAuth token)."
translation to plain lang i think:
"To get a token, send a POST request to the access token URL."

Reply:
rewrote.  now article still has 4 todos, but they all are in the one old, abstruse paragraph.